src/main/java/org/ambraproject/rhino/config/HibernateLoggingInterceptor.java

```diff
@@ -58,6 +59,10 @@ public boolean onSave(Object entity, Serializable id, Object[] state, String[] p
       Comment comment = (Comment) entity;
       String commentCreationMessage = String.format("Comment created. URI: %s ", comment.getCommentUri());
       kafkaEventProducer.send(new ProducerRecord<>("ambra-comment-created", commentCreationMessage));
+    } else if (entity instanceof Flag) {
+      Flag flag = (Flag) entity;
+      String flagCreationMessage = String.format("Comment flagged. URI: %s ", flag.getFlaggedComment().getCommentUri());
+      kafkaEventProducer.send(new ProducerRecord<>("ambra-flag-created", flagCreationMessage));
```

Comment:
Looks like it's time to create a method to do the sending?  As, it looks like they have the same pattern, something like:
```
void sendMessage(String topic, String message);
```